examples/recursion/guest/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Does this example make the `examples/verifier` redundant? Should we remove that from the code base? 

Maybe this example can do what the verifier example does already.